src/en/charms-bundles.md

```diff
@@ -174,6 +174,29 @@ mysql:
       "gui-y": "168"
 ```
 
+## Setting charm configurations options in a bundle
+
+When deploying an application, the charm you use will often support or even
+require specific configuration options to be set. These options can be set in
+a bundle as a simple key addition to the application definition, using the
+configuration key/value pair.
+[See the documentation on application configuration](charms-config.html#discovering-application-configuration-options)
```

Comment:
If you can change the link line to include the ./ that would be great. Like this:
[See the documentation on application configuration](./charms-config.html#discovering-application-configuration-options)